com.xargon.retrodeck.yml

```diff
@@ -0,0 +1,643 @@
+   - --share=ipc
+   - --share=network
+   - --device=all
+   - --filesystem=~/retrodeck:create
+   - --filesystem=/run/media
+   - --allow=multiarch
+   - --talk-name=org.freedesktop.ScreenSaver
+   - --talk-name=org.freedesktop.PowerManagement.Inhibit
+   - --talk-name=org.freedesktop.login1
+   #- --filesystem=host
+   - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  # Yuzu
+   - --filesystem=home:ro
+  # - --filesystem=/run/media:ro
+
+cleanup:
+  # ES-DE
+  - /include
+  - /share/ffmpeg
+  - /lib/cmake
+  #- /lib/debug
+  - /lib/pkgconfig
+  # Yuzu
+  - /include
+  - /bin/glslangValidator
+  - /bin/zip*
+  - /bin/zstd*
+  - /lib/pkg-config
+  - /share/doc
+  - /share/man
+  - /src
+  - '*.a'
+  - '*.la'
+cleanup-commands:
+  # Yuzu
+  - /app/cleanup-BaseApp.sh
+
+modules:
+
+  - name: xmlstarlet
+    config-opts:
+      - --disable-static-libs
+      - --with-libxml-libs-prefix=/usr/lib
+      - --with-libxml-include-prefix=/usr/include/libxml2
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/xmlstar/xmlstarlet-1.6.1.tar.gz
+        sha256: 15d838c4f3375332fd95554619179b69e4ec91418a3a5296e7c631b7ed19e7ca
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .
+          - autoreconf -vfi
+    post-install:
+      - ln -s "xml" "${FLATPAK_DEST}/bin/xmlstarlet" ||:
+    cleanup: ["*"]
+
+  - name: kdialog
+    buildsystem: cmake-ninja
+    sources:
+    - type: git
+      url: https://github.com/KDE/kdialog.git
+      tag: v22.03.80
+
+  # ES-DE
+
+  - name: ffmpeg
+    config-opts:
+    - --disable-static
+    - --disable-programs
+    - --disable-doc
+    - --enable-gpl
+    - --enable-nonfree
+    - --enable-shared
+    - --enable-libvorbis
+    - --enable-libopus
+    - --enable-libfdk-aac
+    - --enable-libvpx
+    - --enable-postproc
+    sources:
+    - type: git
+      url: https://github.com/FFmpeg/FFmpeg.git
+      tag: n5.0
+
+  - name: freeimage
+    no-autogen: true
+    build-options:
+        cxxflags: -std=c++14
+    make-args:
+    - DESTDIR=/app
+    sources:
+    - type: archive
+      url: http://downloads.sourceforge.net/freeimage/FreeImage3180.zip
+      sha256: f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd
+    - type: shell
+      commands:
+      - sed -i 's|-o root -g root ||' ./Makefile.gnu
+      - sed -i 's|/usr|/app|' ./Makefile.gnu
+
+  - name: pugixml
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DBUILD_SHARED_LIBS=on .
+    sources:
+    - type: archive
+      url: https://github.com/zeux/pugixml/releases/download/v1.11.4/pugixml-1.11.4.tar.gz
+      sha256: 8ddf57b65fb860416979a3f0640c2ad45ddddbbafa82508ef0a0af3ce7061716
+
+  - name: emulationstation-de
+    buildsystem: cmake-ninja
+    cleanup:
+      - es-app
+      - es-core
+    sources:
+    - type: git
+      url: https://gitlab.com/leonstyhre/emulationstation-de.git
+      branch: c3fc30cfd5c26195024e0da328349d83af067b9e
```

Comment:
```suggestion
      commit: c3fc30cfd5c26195024e0da328349d83af067b9e
```

Reply:
I was told that I cannot install a flatpak from a flatpak, is that possible?
However in that case I wouldn't know which version of the emulator I will be running.